### PR TITLE
GLTFExporter: Skip single morph animation.

### PR DESCRIPTION
**Pull request by takahirox**
glTF animation can't handle single morph animation in multiple morphs

https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#animations

> A Morph Target animation frame is defined by a sequence of scalars of length equal to the number of targets in the animated Morph Target. Morph Target animation is by nature sparse, consider using Sparse Accessors for storage of Morph Target animation.

then let's skip if animation track is `*.morphTargetInfluences[*]` and encourage merge them instead.

I'm thinking to add `.mergeMorphTargetInfluencesTracks` to `AnimationUtils`.

### CODE REVIEW

File: examples/js/exporters/GLTFExporter.js [hunk 1]
```diff
@@ -1378,6 +1378,16 @@ THREE.GLTFExporter.prototype = {
 
 				if ( trackProperty === PATH_PROPERTIES.morphTargetInfluences ) {
 
+					if ( trackNode.morphTargetInfluences.length !== 1 &&
+						trackBinding.propertyIndex !== undefined ) {
+
+						console.warn( 'THREE.GLTFExporter: Skipping animation track "%s" because ' +
+							'glTF can not handle single morph animation. ' +
+							'Try to merge .morphTargetInfluences elements instead.', track.name );
+						continue;
```
Comment (takahirox) [inline examples/js/exporters/GLTFExporter.js]: Can anyone help make this message cleaner and more straightforward?

Comment (donmccurdy) [reply examples/js/exporters/GLTFExporter.js]: `glTF can not handle single morph animation` is maybe confusing? We could (technically) write `0` for the other targets, and that would be valid glTF, but I don't love that idea... maybe phrase this as `THREE.GLTFExporter: Skipping animation track "%s". Morph target keyframe tracks must target all available morph targets for the given mesh.` Perhaps this is something where the glTF spec should provide an alternative, too.

**Comment by mrdoob** [comment]
Should this be merged?

**Comment by mrdoob** [comment]
@donmccurdy looks good?

**Review by donmccurdy — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**